tests/integration/components/popupOptions-test

```diff
@@ -0,0 +1,93 @@
+import Ember from 'ember';
```

Comment:
What is this file?
It doesn't have any extension and it certainly doesn't look like a test.

It is just a copy of the popup mixin.